README.md

```diff
@@ -49,12 +55,12 @@ Only documents belonging to `tenant-1` will be encrypted. To better understand C
 
 ### (optional) Look at an encrypted index
 
-_Note that we are making this request directly to Elasticsearch (port 9200) so we can see what's actually stored._
+_Note that all queries made with `./query-search-service.sh` are being made directly to your search service. We are using a script to detect if you're running open search or elastic search, but there is no involvement from cloaked search. Since these requests go directly to the search service (port 9200) so we can see what's actually stored._
```

Comment:
```suggestion
_Note that all queries made with `./query-search-service.sh` are being made directly to your search service. We are using a script to detect if you're running OpenSearch or Elasticsearch, but there is no involvement from Cloaked Search. Since these requests go directly to the search service (port 9200), we can see what's actually stored._
```